src/Components/HousingFinderList.jsx

```diff
@@ -0,0 +1,46 @@
+import { List, ListItem, Grid, Typography, Link, } from "@material-ui/core";
+import { Email, HomeWorkOutlined, Call } from "@material-ui/icons";
+import { useResourcesStyles } from "../Styles/useResourcesStyles";
+
+const HousingFinderList = ({ resources }) => {
+  const classes = useResourcesStyles();
+  return (
+    <List>
+      {resources.map((contact, idx) => (
+        <ListItem key={idx} style={{ display: "block" }} divider>
+          <Grid container>
+            <Grid item xs={12} sm={6}>
+              <Typography>
+                <Link href={contact.url} color="initial">
+                  {contact.name}
+                </Link>
+              </Typography>
+              <br />
+              <Typography>{contact.desc}</Typography>
+            </Grid>
+            <Grid item xs={12} sm={6}>
+              <Typography >
+                {contact.address && (
+                  <HomeWorkOutlined className={classes.shelterIcon} />
+                )}
+                {contact.address}
```

Comment:
Consider making these clickable too so people don't need to copy and paste.

Reply:
Link them to a google search card or?

Reply:
Yea or a google maps link. Idk what best practices are for addresses.